### PR TITLE
randomized user agent bug fixes

### PR DESCRIPTION
**Pull request by pbiung**
In line 81: with open('user_agents.txt') as fp:
should be with open(user_agents_file) as fp:

### CODE REVIEW

File: google/__init__.py [hunk 1]
```diff
@@ -78,7 +78,7 @@
 install_folder = os.path.abspath(os.path.split(__file__)[0])
 user_agents_file = os.path.join(install_folder, 'user_agents.txt')
 try:
-    with open('user_agents.txt') as fp:
+    with open('user_agents_file') as fp:
```
Comment (fxfitz) [inline google/__init__.py]: Should this not be: `with open(user_agents_file) as fp:`?

Comment (Bilung) [inline google/__init__.py]: Instead of user_agents.txt we should use  user_agents_file

Comment (fxfitz) [inline google/__init__.py]: Right, but I don't think user_agents_file should be in quotes. We should be using the variable name, not the string.

Comment (Bilung) [inline google/__init__.py]: thanks i correct that

**Comment by fxfitz** [comment]
Actually, it looks like we have two PRs that do the same thing (https://github.com/MarioVilas/google/pull/28).

@MarioVilas: Can you merge one of them and cut a new release? :-)

**Pull request closed without merging**